setup.py

```diff
@@ -5,133 +5,51 @@
 https://github.com/pypa/sampleproject
 """
 
-'''
-to install in development mode, run the following code from command line
-"python setup.py develop"
-'''
-
-# Always prefer setuptools over distutils
-from setuptools import setup, find_packages
-# To use a consistent encoding
+import os
 from codecs import open
-from os import path
+
+from setuptools import find_packages, setup
 
 import versioneer
 
-here = path.abspath(path.dirname(__file__))
+here = os.path.abspath(os.path.dirname(__file__))
+
+# Dependencies.
+with open('requirements.txt') as f:
+    requirements = f.readlines()
+install_requires = [t.strip() for t in requirements]
 
-from pip.req import parse_requirements
-install_reqs = parse_requirements('requirements.txt', session=False)
-reqs = [str(ir.req) for ir in install_reqs]
+with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
 
-# Get the long description from the relevant file
-with open(path.join(here, 'README.md'), encoding='utf-8') as f:
-     long_description = f.read()
-#long_description = ""
 setup(
     name='odm2api',
-
-    # Versions should comply with PEP440.  For a discussion on single-sourcing
-    # the version across setup.py and the project code, see
-    # https://packaging.python.org/en/latest/single_source_version.html
     version=versioneer.get_version(),
-
-    description='A Python-based application programmers interface for the Observations Data converter 2 (ODM2) ',
+    description='Python interface for the Observations Data converter 2 (ODM2)',
```

Comment:
I only reduced the description that was already there, but I guess that we should also fix it to `Model`, right @emiliom?